Mathlib/Geometry/Manifold/Instances/Real.lean

```diff
@@ -378,8 +378,10 @@ instance Icc_smooth_manifold (x y : ℝ) [Fact (x < y)] :
 
 section
 
-instance : ChartedSpace (EuclideanHalfSpace 1) (Icc (0 : ℝ) 1) := by infer_instance
+instance IccZeroOneChartedSpace :
```

Comment:
I just realised: I think the point of these instances is not to be *used* --- but merely to show that they are present. (They are, in fact, subsumed by the instances above.) In other words: I'm not sure if naming these is actually useful. (Adding a comment about this is certainly welcome.)